src/components/TimelineNav/TimelineNav.tsx

```diff
@@ -318,13 +326,19 @@ export const TimelineNav = ({
     }
   };
 
+  const onClick = () => setIsActive(false);
```

Comment:
I think the keyboard focus is getting lost after switching views. We should manually place the focus on the first interactive element on the new view each time.

Reply:
good catch! focus now remains on the last selected nav item

Reply:
We want it to go the other way as well – when you click on a timeline item, it switches to the panel and sets the focus on the back button.

Reply:
This is still unresolved.

Reply:
This is done but could use TD to understand why it only works when it's within a delayed callback.